### ISSUE TITLE
SA1516 doesn't report missing blank line after file scoped namespace

### ISSUE THREAD

**Issue by JakubLinhart** [DotNetAnalyzers/StyleCopAnalyzers#3512]
This code:

```C#
namespace MyNamespace;
public class MyClass
{
}
```

doesn't show SA1516 and SA1513 cannot workaround it.

I would expect that SA1516 is reported for `public class MyClass` like it is for `namespace MyNamespace;` here:

```C#
using System;
namespace MyNamespace;
```

Tested with StyleCop.Analyzers 1.2.0-beta.435.

**Comment by martynas1266** [comment]
Any plans to merge PR and close this issue? We are using file-scoped namespaces in our project and it would be nice to enforce this rule.

**Comment by ldeluigi** [comment]
We too

**Comment by prlcutting** [comment]
Ditto.

**Comment by symbiogenesis** [comment]
bump

**Comment by guillemurano** [comment]
I have the same issue with SA1402 with scoped manespace

**Comment by bjornhellander** [comment]
@guillemurano SA1402 was updated to handle this last year. It was included in 1.2.0-beta.406. What version are you using?

**Comment by guillemurano** [comment]
> @guillemurano SA1402 was updated to handle this last year. It was included in 1.2.0-beta.406. What version are you using?

Excellent, I was on the latest version, works fine with the prerelease
